packages/nimble-components/src/combobox/index.ts

```diff
@@ -57,9 +58,33 @@ export class Combobox
     @attr({ attribute: 'error-visible', mode: 'boolean' })
     public errorVisible = false;
 
+    @observable
+    public region?: AnchoredRegion;
+
+    @observable
+    public controlWrapper?: HTMLElement;
```

Comment:
Can we call this `control` to keep it consistent with the `Select` component?

Reply:
Unfortunately, there's already another element referenced with the name `control`, and the `FoundationCombobox` code uses it.